DOCUMENTATION.md

```diff
@@ -3592,6 +3592,10 @@ The ProcessFixedUpdate method is run in every FixedUpdate method on the interact
 
 The Climbable Grab Attach script is used to mark the object as a climbable interactable object.
 
+### Inspector Parameters
```

Comment:
Do I need to revert the DOCUMENTATION.md change or will that get auto updated? Fixed all the rest.